redhat-developer/app-services-ansible
# Get kafkas

To verify run:

```shell
ansible-galaxy collection build --force
ansible-galaxy collection install <ANSIBLE-COLLECTION-NAME-FROM-PREVIOUS-STEP> -vvv --force
```

just make sure the path to the `tar.gz` file is accurate

Then one can run a playbook of this:

```yaml
---
- name: RHOAS test
  hosts: localhost
  gather_facts: false
  connection: local
  tasks:
  - name: Get kafkas
    rhoas.rhoas.get_kafkas:
      # openshift_offline_token
    register:
      kafkas_resp_obj
  - name: Debug kafkas
    debug:
      msg: "{{ kafkas_resp_obj }}"
```

Which should spit back all kafkas available to the users org
